send_email_pdf/main.py

```diff
@@ -0,0 +1,293 @@
+"""File that creates the visualizations for a report pdf of the media-sentiment findings,
+then uploads the pdf to an s3 bucket and sends an email of the pdf."""
+
+
+import sys
+from os import environ
+from email.mime.multipart import MIMEMultipart
+from email.mime.application import MIMEApplication
+from datetime import datetime
+
+from dotenv import load_dotenv
+from psycopg2.extensions import connection
+from psycopg2 import connect, Error
+import pandas as pd
+import plotly.graph_objects as go
+from xhtml2pdf import pisa
+from boto3 import client
+
+
+PDF_FILE_NAME = "Media-Sentiment.pdf"
+
+
+def join_all_stories_info(conn: connection) -> pd.DataFrame:
+    "Function that joins the stories and story sources SQL tables"
+    query = """SELECT story_id, title, description,url, pub_date, media_sentiment,
+    source_name FROM stories JOIN sources ON stories.source_id = sources.source_id"""
+    with conn.cursor() as cur:
+        cur.execute(query)
+        tuples_list = cur.fetchall()
+    stories_df = pd.DataFrame(tuples_list, columns=[
+        "story_id", "title", "description", "url", "pub_date",
+        "article_sentiment", "source_name"])
+    return stories_df
+
+
+def join_all_reddit_info(conn: connection) -> pd.DataFrame:
+    "Function that joins all reddit sources SQL tables"
+    query = """SELECT * FROM reddit_article"""
+    with conn.cursor() as cur:
+        cur.execute(query)
+        tuples_list = cur.fetchall()
+    reddit_df = pd.DataFrame(tuples_list, columns=["article_id", "domain", "title", "article_url",
+                                                   "url", "re_sentiment_mean", "re_sentiment_st_dev",
+                                                   "re_sentiment_median", "re_vote_score", "re_upvote_ratio", "re_post_comments",
+                                                   "re_processed_comments", "re_created_timestamp"])
```

Comment:
Nit pick: formatting is a bit messy here, could be cleaned up

Reply:
How do you suppose i clean it up?